ArrtModel/Model/ArrSessionManager.cpp

```diff
@@ -904,7 +904,7 @@ void ArrSessionManager::startInspector()
         qInfo(LoggingCategory::renderingSession)
             << tr("Requesting connection to ARR inspector.\n")
             << m_lastProperties;
-        if (const auto async = m_session->ConnectToArrInspectorAsync(m_lastProperties.Hostname.c_str()))
```

Comment:
how come this is not needed anymore?

Reply:
we made a change in client SDKs version 0.1.76 to remove the hostname parameter from this method as ConnectToArrInspectorAsync is session specific. Please see the release notes at https://github.com/Azure/azure-remote-rendering/releases/tag/2020-12-22